src/tests/article.test.js

```diff
@@ -558,7 +558,51 @@ describe('POST /api/v1/articles/{id}/rate', () => {
         if (err) done(err);
         expect(res).have.status(400);
         expect(res).to.be.an('object');
-        expect(res.body.error).to.deep.equal('id must be a number');
+        expect(res.body.error).to.deep.equal('articleId must be a number');
+        done();
+      });
+  });
+});
+
+// getting ratings
+describe('GET /api/v1/articles/{articleId}/rate', () => {
+  it('Should be be able to get ratings', (done) => {
+    chai
+      .request(app)
+      .get(`/api/v1/articles/${articleId}/rate`)
+      .set('Authorization', userToken1)
+      .end((err, res) => {
+        if (err) done(err);
+        expect(res).have.status(200);
+        expect(res).to.be.an('object');
+        done();
+      });
+  });
+  it('Should not be be able to get ratings', (done) => {
```

Comment:
the description if this test is incomplete

Reply:
on it thanks @mkiterian

Reply:
@mkiterian I've Implemented your feedback